### ISSUE TITLE
Prompt to set Min as default browser breaks automated installs

### ISSUE THREAD

**Issue by chepec** [minbrowser/min#882]
[Min 1.12](https://github.com/minbrowser/min/releases/tag/v1.12.0) comes with this new feature:

> You will now be prompted to set Min as your default browser after installing on Linux (by @lee-yunseok)

This is what happened when I upgraded (Ubuntu 18.04):
```
$ sudo apt full-upgrade 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Calculating upgrade... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
1 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Do you want to continue? [Y/n] 
Setting up min (1.12.0-1) ...
Would you like to set Min as your default web browser? [Y/n]
```

This implementation is a very unusual way to do it, and **will break** all kinds of automation built on `apt`. For example, Ansible playbooks will choke and error out on this prompt.

Would you consider accomplishing this feature in a more traditional way, as a popup window issued by the Min GUI, for example?

**Comment by PalmerAL** [comment]
Hmm, that's not good. Assuming there isn't a more standard way to do this after installation (is there?), I'd be fine with just making this a message telling you how to run the command yourself. @lee-yunseok does that sound OK to you?

**Comment by lee-yunseok** [comment]
Sorry for late reply. I didn't expect this issue. Then remove postinst script.

**Comment by PalmerAL** [comment]
I don't think the script needs to be removed entirely; would something like this work?

```bash
#!/bin/sh
set -e

if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ]; then
    update-alternatives --install /usr/bin/x-www-browser \
        x-www-browser /usr/bin/min 80
    echo "Run sudo xdg-settings set default-web-browser min.desktop && sudo update-alternatives --config x-www-browser to set Min as your default browser."
fi
```

**Comment by lee-yunseok** [comment]
In these days I'm using RPi only, so I can't test on Ubuntu or other Linux distro.

**Comment by lee-yunseok** [comment]
Please check the pull request #886

**Comment by christianrauch** [comment]
The `min` package should really not ask interactively to become the default browser. This breaks many `apt`-based GUI tools and the dpkg database. Interactive package installation is an annoying thing in any case.

What is wrong with asking the user about making it the default on the first start? This is how Firefox, Chrome and Vivaldi are doing it.

**Comment by PalmerAL** [comment]
Fixed by #896.
